spec/style.scss

```diff
@@ -1,4 +1,12 @@
+@import "../components/commons";
+
 .app {
```

Comment:
I think this class belongs to the `commons` file. You have to think that every client app using React Toolbox would need to assign this styles to the app root node in order to have properly working Drawer and Dialog components. Why moving it here?